apps/site/lib/site_web/controllers/transit_near_me_controller.ex

```diff
@@ -29,6 +29,7 @@ defmodule SiteWeb.TransitNearMeController do
   end
 
   defp assign_location(conn) do
+    # Why do we check for location_fn?  It's never assigned
```

Comment:
Hmm! I think it’s a convention that arose via certain developers… there are similar examples of passing in functions with defaults like this throughout the codebase. Sometimes it seems like a helpful way to easily enable alternate implementations i.e. from one page we want function A but from a different page we want to use function B and from the test case we will pass in mocked function C etc.